19.md

```diff
@@ -7,9 +7,9 @@ LUD-19: Pay link discoverable from withdraw link.
 
 ## Merging "pay" and "withdraw" related to a same `SERVICE` in a `WALLET`
 
-LNURL-withdraw may contain a `payLink` in its JSON response. This is done to recognize a fact that `SERVICE` may want to allow both deposits and withdrawals to user account and as such make a static payment link easily discoverable from withdraw link.
+LNURL-withdraw may contain a `payLink` in its JSON response. This is done to recognize a fact that `SERVICE` may want to allow both deposits and withdrawals to user account and as such, make a static payment link easily discoverable from withdraw link.
```

Comment:
I'm not sure this correction is actually grammatically correct?
The explanation follows the "as such", putting a comma seems off.